torchvision/utils.py

```diff
@@ -115,7 +115,7 @@ def norm_range(t, value_range):
 def save_image(
     tensor: Union[torch.Tensor, List[torch.Tensor]],
     fp: Union[Text, pathlib.Path, BinaryIO],
-    normalize: bool = False,
+    normalize: Union[bool, str] = 'deprecated',
```

Comment:
Good catch. Because here we also include a `**kwargs`, we could completely remove this line:
```suggestion
```